go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/questdb/go-questdb-client/v2
+module github.com/questdb/go-questdb-client/v3
 
 go 1.17
 
```

Comment:
Go 1.17 is quite old, so it's not a big deal if we upgrade minimal Go version. It'll be much easier to do that now since we're going to release a major version.